io-engine/src/lvs/lvs_store.rs

```diff
@@ -409,41 +437,64 @@ impl Lvs {
         }
     }
 
-    /// Create a pool on base bdev
-    pub async fn create(
-        name: &str,
-        bdev: &str,
-        uuid: Option<String>,
-        cluster_size: Option<u32>,
+    /// Converts floating point metadata reservation ratio into SPDK's format.
+    fn mdp_ratio(args: &PoolArgs) -> Result<u32, LvsError> {
+        if let Some(h) = args.md_args.as_ref().and_then(|p| p.md_resv_ratio) {
+            if h > 0.0 {
```

Comment:
Should we better ensure this is `>1.0` ? Otherwise it indicates shrinkage.

Reply:
Nope; it means reserve less space for metadata.
E.g., 0.5 means allocate 1 MD page per 2 segments.
0.25 means allocate 1 MD page per 4 segments.